edgecast/resources/originv3/origin_httplarge.go

```diff
@@ -120,8 +120,39 @@ func ResourceOriginGroupUpdate(
 	d *schema.ResourceData,
 	m interface{},
 ) diag.Diagnostics {
-	// Not implemented
-	return diag.Diagnostics{}
+	config, ok := m.(internal.ProviderConfig)
+	if !ok {
+		return diag.Errorf("failed to load configuration")
+	}
+
+	svc, err := buildOriginV3Service(config)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	grpID, err := helper.ParseInt64(d.Id())
```

Comment:
We're parsing to int64 but below casting to int32 - perhaps we can create a `helper.ParseInt32` method?